src/screenshotbot/compare.lisp

```diff
@@ -377,13 +397,17 @@
                 for comparison-image = (util:copying (image-comparison-job)
                                          (nibble ()
                                            (prepare-image-comparison image-comparison-job)))
+
+                for image-campare-ret = (is-image-similiar before after)
+
                 collect
                 <div class= "image-comparison-wrapper" >
                 <h3>,(screenshot-name before)</h3>
+                ,(when (= image-campare-ret 1)
```

Comment:
this will still create a title but not have the image (no check on L405 for instance).  The `loop` construct has an `if` argument that you can use instead. So on L401 you can say `if (= (is-image-similar before after) 1)`

Reply:
After a short conversation with my team, we might need the title in this case, I am whipping up another commit to merge `is-image-similiar` and make the title has the **same** info.

Reply:
We might need the title in this case, I just add some hint on it.